### PR TITLE
Add provider flag for a policy global manager env

### PR DESCRIPTION
**Pull request by asarfaty**

### CODE REVIEW

File: nsxt/provider.go [hunk 1]
```diff
@@ -145,6 +146,12 @@ func Provider() terraform.ResourceProvider {
 				Description: "Enforcement Point for NSXT Policy",
 				DefaultFunc: schema.EnvDefaultFunc("NSXT_POLICY_ENFORCEMENT_POINT", "default"),
 			},
+			"policy_global_manager": {
```
Comment (annakhm) [inline nsxt/provider.go:149]: I'd suggest to call it just "global_manager" - Madhu, what do you think?

Comment (madhukark) [reply nsxt/provider.go:149]: I agree, just "global_manager" is good.

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -123,6 +123,18 @@ provider "nsxt" {
...
 ```
 
+### Policy Global Manager Example
+
+```hcl
+provider "nsxt" {
+  host                  = "192.168.110.41"
+  username              = "admin"
+  password              = "default"
+  policy_global_manager = true
```
Comment (madhukark) [inline website/docs/index.html.markdown:133]: global_manager

File: website/docs/index.html.markdown [hunk 3]
```diff
@@ -177,6 +189,8 @@ The following arguments are used to configure the VMware NSX-T Provider:
 * `enforcement_point` - (Optional) Enforcement point, mostly relevant for policy
   data sources. For VMC environment, this should be set to `vmc-enforcementpoint`.
   For on-prem deployments, this setting should not be specified.
+* `policy_global_manager` - (Optional) True if this is a policy global manager
```
Comment (madhukark) [inline website/docs/index.html.markdown:192]: Here too, just 'global_manager'. Also, can change the string 'policy global manager' to 'Global Manager'

File: website/docs/index.html.markdown [hunk 4]
```diff
@@ -127,10 +127,10 @@ provider "nsxt" {
 
 ```hcl
 provider "nsxt" {
-  host                  = "192.168.110.41"
-  username              = "admin"
-  password              = "default"
-  policy_global_manager = true
+  host           = "192.168.110.41"
+  usernam        = "admin"
```
Comment (enhaocui) [inline website/docs/index.html.markdown:131]: Oops, typo?

**Pull request merged**